packages/sdk/src/analytics/useAnalyticsEvent.ts

```diff
@@ -4,7 +4,7 @@ import { ANALYTICS_EVENT_TYPE, unwrap } from './wrap'
 import type { UnknownEvent } from './wrap'
 
 export const useAnalyticsEvent = <T extends UnknownEvent = UnknownEvent>(
-  handler: (event: T) => unknown
+  handler: (event: T, isEcommerceEvent?: boolean) => unknown
```

Comment:
It looks like this change is unnecessary. as we are not passing the isEcommereceEvent down to the handler on lin 16 neither reading this on `packages/core/src/sdk/analytics/index.tsx`


```suggestion
  handler: (event: T) => unknown
```